web-bundle/src/main/java/com/graphhopper/http/GraphHopperManaged.java

```diff
@@ -61,26 +69,33 @@ public GraphHopperManaged(CmdArgs configuration, ObjectMapper objectMapper) {
         } else {
             graphHopper = new GraphHopperOSM(landmarkSplittingFeatureCollection).forServer();
         }
-        String spatialRuleLocation = configuration.get("spatial_rules.location", "");
-        if (!spatialRuleLocation.isEmpty()) {
+        String spatialRuleBordersDirLocation = configuration.get("spatial_rules.borders_directory", "");
```

Comment:
Can we throw an exception if property `spatial_rules.location` is still used and inform about the new property?

Reply:
Sure.